tutorial/source/bo.ipynb

```diff
@@ -164,7 +166,9 @@
     "    X = torch.cat([gpmodel.X, x_new]) # incorporate new evaluation \n",
     "    y = torch.cat([gpmodel.y, y])     \n",
     "    gpmodel.set_data(X, y)\n",
-    "    gpmodel.optimize()  # optimize the GP hyperparameters using default settings"
+    "    # optimize the GP hyperparameters using Adam optimizer\n",
+    "    optimizer = optim.Adam(gpmodel.parameters())\n",
```

Comment:
i'm surprised you use adam with default parameters; i've usually found this is too much momentum in this setting

Reply:
@martinjankowiak this is just to replicate the old behaviour. I agree that we should use another cheaper one (such as LBFGS) or with a higher `lr`. If you want a change, I'll try it after debugging `oed`.